### PR TITLE
Feature/update readme to 0.3

### PR DESCRIPTION
**Pull request by barweiss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,14 @@
...
 The `seekret.apitest` package contains runtime functions and tools intended to ease API testing.
 
-The `seekret.apitest` package is not used directly, but is referenced within tavern tests generated by Seekret.
+Seekret uses this library in automatically generated tests to help the readability of the generated tests, and implement
+common functionalities.
 
 ## Quickstart
 
-First, install tavern and seekret.apitest: `pip install tavern seekret.apitest`
+First, install seekret.apitest::
```
Comment (asdftamir) [inline README.md]: ::

File: README.md [hunk 2]
```diff
@@ -2,11 +2,14 @@
...
+
+    pip install tavern seekret.apitest
```
Comment (asdftamir) [inline README.md]: OMG this is the forbidden word

File: README.md [hunk 3]
```diff
@@ -17,173 +20,137 @@ Your test directory should look like this:
...
-## Understanding the configuration and generated tests
+Use the `--log-cli-level INFO` option to show outgoing requests and incoming responses live as they occur during the
+test.
 
-Tavern tests generated by Seekret use common variables which are expected to be set using an external configuration.
-This currently includes the host of the target servers and the authorization settings.
+## Run profile
 
-Example generated test file:
+Each test run requires a run profile. By default, Seekret takes reads the run profile from the `run-profile.yaml` file
```
Comment (asdftamir) [inline README.md]: takes reads

File: README.md [hunk 4]
```diff
@@ -17,173 +20,137 @@ Your test directory should look like this:
...
+
+```python
+import seekret.apitest
+
+
+def test_post_channels_post_messages(seekret: seekret.apitest.Context):
+    # Stage 1: POST /api/channels
+    with seekret.stage(method='POST', path='/api/channels') as request:
+        response = request(json={
+            'name': 'fuGfYxcrgCSsVBMLlceEpLrGPT'
```
Comment (asdftamir) [inline README.md]: give name a better value

File: README.md [hunk 5]
```diff
@@ -17,173 +20,137 @@ Your test directory should look like this:
...
+        })
+        assert response.status_code == 201
+
+        carry_0_responseBody_data_channel_id = response.search('json.data.channel.id')
+
+    # Stage 2: POST /api/channels/{channel_id}/messages
+    with seekret.stage(method='POST', path='/api/channels/{channel_id}/messages') as request:
+        response = request(json={
+            'message': 'YdKQyxQAVPenmYCQghMhtctDywXWXOrZJDakCrQYoIa'
```
Comment (asdftamir) [inline README.md]: give message a better value

File: README.md [hunk 6]
```diff
@@ -17,173 +20,137 @@ Your test directory should look like this:
...
+import pytest
+import seekret.apitest
+
+
+@pytest.fixture
+def channel(seekret: seekret.apitest.Context):
+    # Setup stage: happens before the test.
+    with seekret.stage(method='POST', path='/api/channels') as request:
+        response = request(json={
+            'name': 'fuGfYxcrgCSsVBMLlceEpLrGPT'
```
Comment (asdftamir) [inline README.md]: better name

File: README.md [hunk 7]
```diff
@@ -17,173 +20,137 @@ Your test directory should look like this:
...
+    # Teardown stage: happens after the test.
+    with seekret.stage(method='DELETE', path='/api/channels/{channel_id}'):
+        response = request(path_params={
+            'channel_id': channel_id
+        })
+        assert response.status_code == 200
+
+
+def test_post_messages_delete_messages(seekret: seekret.apitest.Context, channel):
+    # The 'channel' parameter has the value yielded from the 'channel' fixture.
```
Comment (asdftamir) [inline README.md:154]: expand the example here, do they use `channel` or `channel_id`

**Review by asdftamir — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (barweiss) [reply README.md]: `::` starts a code block

File: README.md (hunk 6 above)
Comment (barweiss) [reply README.md]: Fixed

File: README.md (hunk 5 above)
Comment (barweiss) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (barweiss) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (barweiss) [reply README.md]: 😱

File: README.md (hunk 3 above)
Comment (barweiss) [reply README.md]: Fixed

File: README.md (hunk 7 above)
Comment (barweiss) [reply README.md:154]: Fixed

**Review by asdftamir — APPROVED** [review]

**Pull request merged**